code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -509,4 +520,27 @@
 
 // No binary for pAIs.
 /mob/living/silicon/pai/binarycheck()
-	return 0
\ No newline at end of file
+	return 0
+
+// Handle being picked up.
+
+/mob/living/silicon/pai/get_scooped(var/mob/living/carbon/grabber)
+	var/obj/item/weapon/holder/H = ..()
+	if(!istype(H))
+		return
+	H.icon_override = 'icons/mob/in-hand/paiheld.dmi'//I have these in diffrent DMI so i am overriding
+	H.icon_state = "pai-[icon_state]"
+	grabber.put_in_active_hand(H)//for some reason unless i call this it dosen't work
+	grabber.update_inv_l_hand()
+	grabber.update_inv_r_hand()
+
+	return H
+
+/mob/living/silicon/pai/MouseDrop(atom/over_object)
+	var/mob/living/carbon/H = over_object
+	if(!istype(H) || !Adjacent(H)) return ..()
+	if(H.a_intent == "help")
+		get_scooped(H)
+		return

```

Comment:
This return is a pointless line, it does absolutely nothing. Which.. is pretty much standard for bay12 code.